x-pack/test/security_solution_cypress/cypress/e2e/investigations/timeline_templates/creation.cy.ts

```diff
@@ -8,97 +8,90 @@
 import { getTimeline } from '../../../objects/timeline';
 
 import {
-  FAVORITE_TIMELINE,
   LOCKED_ICON,
-  NOTES,
-  NOTES_TAB_BUTTON,
-  NOTES_TEXT_AREA,
   PIN_EVENT,
-  TIMELINE_DESCRIPTION,
   TIMELINE_FLYOUT_WRAPPER,
   TIMELINE_QUERY,
   TIMELINE_TITLE,
+  TIMELINE_DATE_PICKER_CONTAINER,
 } from '../../../screens/timeline';
-import {
-  TIMELINES_DESCRIPTION,
-  TIMELINES_PINNED_EVENT_COUNT,
-  TIMELINES_NOTES_COUNT,
-  TIMELINES_FAVORITE,
-} from '../../../screens/timelines';
+import { TIMELINES_DESCRIPTION, TIMELINES_FAVORITE } from '../../../screens/timelines';
 import { createTimeline } from '../../../tasks/api_calls/timelines';
 import { deleteTimelines } from '../../../tasks/api_calls/common';
 
 import { login } from '../../../tasks/login';
 import { visit } from '../../../tasks/navigation';
 import { openTimelineUsingToggle } from '../../../tasks/security_main';
 import {
-  addDescriptionToTimeline,
   addFilter,
   addNameToTimelineAndSave,
-  addNotesToTimeline,
   clickingOnCreateTemplateFromTimelineBtn,
   closeTimeline,
   createNewTimelineTemplate,
   expandEventAction,
   markAsFavorite,
-  openTimelineTemplateFromSettings,
+  openTimelineTemplate,
   populateTimeline,
+  addNameAndDescriptionToTimeline,
+  openTimelineTemplatesTab,
 } from '../../../tasks/timeline';
-import { openTimeline, waitForTimelinesPanelToBeLoaded } from '../../../tasks/timelines';
+import {
+  updateTimelineDates,
+  showStartEndDate,
+  setStartDate,
+  setEndDateNow,
+} from '../../../tasks/date_picker';
+import { waitForTimelinesPanelToBeLoaded } from '../../../tasks/timelines';
 
 import { TIMELINES_URL } from '../../../urls/navigation';
 
-// FLAKY: https://github.com/elastic/kibana/issues/165661
 describe('Timeline Templates', { tags: ['@ess', '@serverless'] }, () => {
   beforeEach(() => {
     login();
     deleteTimelines();
     cy.intercept('PATCH', '/api/timeline').as('timeline');
   });
 
-  it.skip('Creates a timeline template', () => {
+  it('Creates a timeline template', () => {
+    createTimeline(getTimeline());
```

Comment:
Why do we need to create a regular timeline before creating a template?

Reply:
Nice catch. I must've copy pasted it from the other test. Fixed in https://github.com/elastic/kibana/pull/173206/commits/2f23f20e15554b8fe401dba01cfd2c9c29826269